docs/source/deploying/prefixed-root.md

```diff
@@ -0,0 +1,68 @@
+---
+html_meta:
+  "description": "Prefix (non-root) deployment for Volto"
+  "property=og:description": "Prefix (non-root) deployment for Volto"
+  "property=og:title": "Prefixed (non-root) deployment"
+  "keywords": "Volto, deployment"
+---
+
+# Prefixed (non-root) deployment
+
+If you're integrating a Volto website with another existing website, you may
+need to run Volto on a "folder" inside that website, rather than
+Volto directly serving the root folder.
```

Comment:
```suggestion
If you're integrating a Volto website with another existing website, you may need to run Volto on a virtual path inside that website instead of the root path.
```